apcd-cms/src/apps/submission_form/templates/submission_form/submission_form.test.html

```diff
@@ -0,0 +1,2965 @@
+
+                <option value="IA - Iowa">IA - Iowa</option>
+
+                <option value="KS - Kansas">KS - Kansas</option>
+
+                <option value="KY - Kentucky">KY - Kentucky</option>
+
+                <option value="LA - Louisiana">LA - Louisiana</option>
+
+                <option value="ME - Maine">ME - Maine</option>
+
+                <option value="MD - Maryland">MD - Maryland</option>
+
+                <option value="MA - Massachusetts">MA - Massachusetts</option>
+
+                <option value="MI - Michigan">MI - Michigan</option>
+
+                <option value="MN - Minnesota">MN - Minnesota</option>
+
+                <option value="MS - Mississippi">MS - Mississippi</option>
+
+                <option value="MO - Missouri">MO - Missouri</option>
+
+                <option value="MT - Montana">MT - Montana</option>
+
+                <option value="NE - Nebraska">NE - Nebraska</option>
+
+                <option value="NV - Nevada">NV - Nevada</option>
+
+                <option value="NH - New hampshire">NH - New hampshire</option>
+
+                <option value="NJ - New jersey">NJ - New jersey</option>
+
+                <option value="NM - New mexico">NM - New mexico</option>
+
+                <option value="NY - New york">NY - New york</option>
+
+                <option value="NC - North carolina">NC - North carolina</option>
+
+                <option value="ND - North dakota">ND - North dakota</option>
+
+                <option value="MP - Northern Mariana Islands">
+                  MP - Northern Mariana Islands
+                </option>
+
+                <option value="OH - Ohio">OH - Ohio</option>
+
+                <option value="OK - Oklahoma">OK - Oklahoma</option>
+
+                <option value="OR - Oregon">OR - Oregon</option>
+
+                <option value="PA - Pennsylvania">PA - Pennsylvania</option>
+
+                <option value="PR - Puerto rico">PR - Puerto rico</option>
+
+                <option value="RI - Rhode island">RI - Rhode island</option>
+
+                <option value="SC - South Carolina">SC - South Carolina</option>
+
+                <option value="SD - South Dakota">SD - South Dakota</option>
+
+                <option value="TN - Tennessee">TN - Tennessee</option>
+
+                <option value="TX - Texas">TX - Texas</option>
+
+                <option value="UT - Utah">UT - Utah</option>
+
+                <option value="VT - Vermont">VT - Vermont</option>
+
+                <option value="VA - Virginia">VA - Virginia</option>
+
+                <option value="VI - Virgin Islands">VI - Virgin Islands</option>
+
+                <option value="WA - Washington">WA - Washington</option>
+
+                <option value="WV - West Virginia">WV - West Virginia</option>
+
+                <option value="WI - Wisconsin">WI - Wisconsin</option>
+
+                <option value="WY - Wyoming">WY - Wyoming</option>
```

Comment:
**New Ticket.** Not now (cuz today). Make this available to any other custom app. I.e. Add it to Core-CMS.